CONTRIBUTORS

```diff
@@ -71,7 +71,7 @@ Kevin C. Zimmerman <kevinczimmerman@gmail.com>
 Kirill Motkov <motkov.kirill@gmail.com>
 Konstantin Shaposhnikov <k.shaposhnikov@gmail.com>
 Leonid Kneller <recondite.matter@gmail.com>
-Lyron Winderbaum <lyron.winderbaum@student.adelaide.edu.au>
+Lyron Winderbaum <lyron.winderbaum@student.adelaide.edu.au> <armadilloa16@gmail.com>  <lyron.winderbaum@uwa.edu.au>
```

Comment:
Double space between gmail and uwa address.

Reply:
... this is why I love go fmt

Reply:
Indeed